components/service/sync-logins/src/main/java/mozilla/components/service/sync/logins/Types.kt

```diff
@@ -4,16 +4,44 @@
 
 package mozilla.components.service.sync.logins
 
-import mozilla.components.concept.sync.SyncAuthInfo
+import mozilla.components.concept.storage.Login
 
 /**
- * Conversion from a generic AuthInfo type into a type 'logins' lib uses at the interface boundary.
+ * Conversion from a generic [Login] into its richer comrade within the 'logins' lib.
  */
-fun SyncAuthInfo.into(): SyncUnlockInfo {
-    return SyncUnlockInfo(
-        kid = this.kid,
-        fxaAccessToken = this.fxaAccessToken,
-        syncKey = this.syncKey,
-        tokenserverURL = this.tokenServerUrl
+fun Login.toNativeLogin(): mozilla.appservices.logins.Login {
```

Comment:
I should have brought this up before, but I'm not sure about the `Native` in the name here - it kinda reads like you mean `toAndroidLogin` or `toIOSLogin`. Maybe the word `Storage` instead? Or maybe we just let it go and see what the Android team says - they might be fine with it. Just food for thought...

Reply:
Yeah I can see how it would seem flipped for the other way. I think maybe `toRustLogin` might make it explicitly clear? I like `toStorageLogin` aswell but yeah this might be something the Android team might chime in in terms of how they think about the separation.

Reply:
We certainly use `Rust` as a name prefix in other places currently in similar cases, so I wouldn't mind `toRustLogin`; that's pretty clear.

Reply:
Although, `toDatabaseLogin` better aligns with your class name. Either way, I have no strong preference.

Reply:
I definitely like `toDatabaseLogin` that actually feels the most "natural". Since we're more or less dealing with a thin kotlin wrapper on the lib side of things (and that interfaces with the rust). Thanks for the suggestion!